### PR TITLE
Add sample consumer in channels sample

### LINKED ISSUE #248: Saturn Channels - No tests, sample doesn't have a sample client
The example given just sends a ping out to connected clients. Nowhere is there an example oh consuming it from JavaScript or elsewhere. One of the problems with no consumer, is there's no clear way to address a handle based on the sample, the [documentation ](https://github.com/SaturnFramework/Saturn/blob/master/src/Saturn/Channels.fs#L169) does show it. I'm working on a PR for this. I've got a simple static html page that consumes this in my fork.

### PR DESCRIPTION
**Pull request by ImaginaryDevelopment**
closes #248.

This pull request will add a sample static html page that calls into the channel topics to serve as a more fleshed out sample

### CODE REVIEW

**Comment by Krzysztof-Cieslak** [comment]
Hey, 

Thanks a lot for PR. Is it complete? If so please remove `[WIP]` from the title and then I'll review and merge it.

**Comment by ImaginaryDevelopment** [comment]
no, the handles aren't triggering, not even sure how to do that, can you assist?

File: sample/ChannelsSample/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            p.style.wordWrap = 'break-word';
+            p.innerHTML = msg;
+            domElement.appendChild(p);
+        }
+    }
+
+    let doSend = (ws,writer,msg) => {
+        let display = typeof msg == "string" ? msg || typeof msg == "number" : JSON.stringify(msg);
+        writer('Sending: ' + display);
+        ws.send(msg);
```
Comment (Krzysztof-Cieslak) [inline sample/ChannelsSample/index.html:33]: `ws.send` doesn't support automatic serialization - you need to do that before. In current version message sent to the server was a string `[object Object]` which obviously was not parsed successfully. 

```suggestion
        ws.send(display);
```

**Comment by Krzysztof-Cieslak** [comment]
Thanks a lot for the PR! ❤️

**Pull request merged**